test_runner/src/main/kotlin/ftl/args/IosArgs.kt

```diff
@@ -18,15 +18,13 @@ import ftl.args.yml.IosFlankYml
 import ftl.args.yml.IosGcloudYml
 import ftl.args.yml.IosGcloudYmlParams
 import ftl.args.yml.YamlDeprecated
-import ftl.util.loadFile
 import ftl.cli.firebase.test.ios.IosRunCommand
 import ftl.config.Device
 import ftl.config.FtlConstants
 import ftl.ios.IosCatalog
 import ftl.ios.Xctestrun
 import ftl.run.status.asOutputStyle
-import ftl.util.FlankFatalError
-import ftl.util.FlankTestMethod
+import ftl.util.*
```

Comment:
I think we have a lint rule against global imports?

Reply:
Yes, but I not enabling check when pull request is draft. Tell me please if I should commit only checked code even if pull request is a draft. I want make all right :)

Reply:
I think if you adjust your IDE settings then the problem will go away.

It's fine to have code that's not ready yet for a draft PR. 🙂